### ISSUE TITLE
Skater tandem names don't match their associated Property names

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/energy-skate-park-basics#358]
Noted while working on https://github.com/phetsims/phet-io/issues/661.

Skater has many tandem names that don't match their associated Property names (which is supposed to be the case, by convention.) This made it significantly more difficult to convert.

**Comment by pixelzoom** [comment]
All such Properties are marked with a TODO item that references this issue.

**Comment by samreid** [comment]
Marked for phet-io meeting to decide on the variable names.

**Comment by samreid** [comment]
We decided to use shorter but not cryptic names, like `parametricPosition` and `parametricSpeed`.  @samreid should take care of this.

**Comment by samreid** [comment]
Fixed above and tested for about 5 minutes.  This was an invasive change and possible that I've missed some situations.  @ariel-phet I recommend that this be more thoroughly tested by the QA team before we publish another version out of master.  It can be tested directly from phettest, or we can create a dev publication for testing.  Reassigned to @ariel-phet for feedback.

**Comment by ariel-phet** [comment]
@samreid yes, please create a dev version, also create a test task with some context description (and perhaps some things that might be good to look out for). Please assign the test task to me at first.

**Comment by marlitas** [comment]
This will be tested as part of the upcoming region and culture publication.

**Comment by marlitas** [comment]
Ahhh I did not know this, thanks for the heads up @arouinfar. @kathy-phet this sounds like we will not include phet-io work for region and culture publications then?

**Comment by marlitas** [comment]
PhET-iO brand is not included in the upcoming region and culture publication. Marking as deferred.
